module/VuFind/src/VuFind/ILS/Driver/KohaILSDI.php

```diff
@@ -1244,10 +1245,11 @@ public function getMyHolds($patron)
             $holdLst[] = [
                 'id'       => $this->getField($hold->{'biblionumber'}),
                 'location' => $this->getField($hold->{'branchname'}),
-                // FIXME: require exposure of reserves.expirationdate
-                'expire'   => "N/A",
-                'create'   => date_format(
-                    new \DateTime($this->getField($hold->{'reservedate'})), "m/d/Y"
+                'expire'   => $this->displayDate(
+                    $this->getField($hold->{'expirationdate'})
```

Comment:
Did the FIXME comment you removed refer to something from the old DB-driven version of the code and thus no longer apply? Is there any chance the expiration date will not be set and that we need some isset validation here? (I imagine it's probably fine -- just asking to be on the safe side).

Reply:
I Suspect the FIXME was related to an old version of Koha's ILS-DI interface. But I'll add an isset default to the expiration date field of N/A, just to be safe.